docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,156 @@
+# Smart Contract Upload
+
+This tutorial covers how to create a circuit on a splinter network between 
+two nodes and upload a smart contract via the circuit.
+
+## Prerequisites
+
+* A splinter network with at least two nodes running. The tutorial will refer
+  to the two nodes as "alpha" and "beta" respectively
+* The scabbard command line tool installed
+* The splinter command line tool installed
+* A compiled smart contract. Check [here](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/application_developer_guide.html#) for details on how to create and compile a smart contract
+
+## Procedure
+
+1. Create a new circuit
+
+    a. Define the circuit
+    
+    Below is a sample circuit file to upload
+
+    ```
+    # circuit.yaml
+
+    circuit_id: my-circuit
+    roster:
+      - service_id: scabbard-service-a
+        service_type: scabbard
+        allowed_nodes:
+        - alpha-node-000
+        arguments:
+        - ["admin_keys", "[\"<public_key>\"]"]
```

Comment:
Can you explain why admin_keys are required and specify that the following scabbard commands **must** be used with this admin_key.

Reply:
Updated with explanation